packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -123,6 +128,70 @@ class EmbeddedContentConsumer extends React.Component {
+
+    // Check if the content is scrollable
+    const isContentScrollable = () => {
+      const doc = this.xfcFrame?.iframe?.contentWindow?.document;
+      return (doc.documentElement.scrollHeight > doc.documentElement.clientHeight
+        || doc.body.scrollHeight > doc.body.clientHeight
+        || doc.documentElement.scrollWidth > doc.documentElement.clientWidth
+        || doc.body.scrollWidth > doc.body.clientWidth);
+    };
+
+    // Check if the iframe has `scrolling` attribute set or not
+    // The default `scrolling` attribute is `auto`.
+    const scrollingEnabled = () => (this.xfcFrame?.iframe?.getAttribute('scrolling') !== 'no');
+
+    // Event listener and callback function for when `load` is completed for the content in the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('load', () => {
+      this.hasInteractableElement = [...this.xfcFrame?.iframe?.contentWindow?.document.body.querySelectorAll(`${interactableElementSelector}`)].some(
+        (element) => !element.hasAttribute('disabled')
+          && !element.getAttribute('aria-hidden')
+          && !!(element.offsetWidth || element.offsetHeight || element.getClientRects().length)
+          && window.getComputedStyle(element).visibility !== 'hidden'
+          && element.closest('[inert]') === null,
+      );
+
+      if (scrollingEnabled() && isContentScrollable() && this.hasInteractableElement === false) {
+        // Set tabIndex="0" so focus can go into the document when
+        // using tab key when scrolling is enabled
+        this.xfcFrame.iframe.contentWindow.document.body.tabIndex = 0;
+      }
+    });
+
+    // Event listener and callback function for `resize` event of the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('resize', () => {
+      if (scrollingEnabled() && isContentScrollable() && this.hasInteractableElement === false) {
+        // Set tabIndex="0" so focus can go into the document when
+        // using tab key when scrolling is enabled
+        this.xfcFrame.iframe.contentWindow.document.body.tabIndex = 0;
+      } else if (this.xfcFrame?.iframe?.contentWindow?.document.body.getAttribute('tabIndex') === '0') {
+        this.xfcFrame.iframe.contentWindow.document.body.removeAttribute('tabIndex');
+      }
+    });
+
+    // Event listener and callback function for `focus` event is in the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('focus', () => {
+      if (this.hasInteractableElement === true || !isContentScrollable()) {
+        return;
+      }
+
+      if (scrollingEnabled() && isContentScrollable() && this.hasInteractableElement === false) {
+        this.xfcFrame.iframe.className = cx('iframe-focus-style');
+      }
+    }, true);
+
+    // Event Listener and callback function for `blur` event in the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('blur', () => {
+      this.xfcFrame.iframe.removeAttribute('class');
```

Comment:
You wouldn't removed the entire class attribute.  You would only remove the class that you added.

Reply:
Yeah, I was using that `classList.remove()` but that only removes the value and not the attribute `class`.

Reply:
Let me update to use this method https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove instead. That seems safer than accidentally removing other keys.

Reply:
https://github.com/cerner/terra-framework/pull/1874/commits/b53056bf58a6fd60d158c1ff8ec4609a75d99f4a#diff-77b1c1b39ad95249bd3956ef509a0995b4687f8c94ba87a3110f3e6b1ce8ccfeR193-R203